docs/use-the-network/coverage-mapping/mappers-quickstart.mdx

```diff
@@ -0,0 +1,122 @@
+---
+id: mappers-quickstart
+hide_title: true
+sidebar_label: Mappers Quickstart
+slug: /use-the-network/coverage-mapping/mappers-quickstart
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+## Mappers Quickstart
+
+This guide will show you step by step how to contribute data to the Mappers
+project with an off the shelf tracker. This guide will use the
+[Dragino LGT92](https://www.dragino.com/products/lora/item/142-lgt-92.html), but
+any LoRaWAN compatible tracker will work.
+
+### Steps
+
+1. Create new Label and HTTP integration with Mappers API ingest endpoint.
+2. Create Function Decoder to decode device payload and attach to Label created
+   in first step.
+3. Attach Label to tracking device and verify data is being sent successfully.
+
+### 1. Create HTTP Integration
+
+To start, we'll create a new HTTP integration in Console. Navigate to the
+Integrations page using the left navigation and then select the HTTP
+integration.
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/console/integrations/integrations-select-http.png"
+  )}
+/>
+
+Next we'll fill in the details, see instructions and image below.
+
+1. Enter the Mappers API Ingest endpoint
+   `https://mappers.helium.wtf/api/v1/ingest`
+2. Enter the name for this Integration: _Mappers Integration_
+3. Create and apply a Label for this Integration, we've used the same name as
+   the integration.
+4. Finally, click _Create Integration_ to complete.
+
+<img
+  src={useBaseUrl(
+    "img/use-the-network/coverage-mapping/mappers-quickstart-create-http-integration.png"
+  )}
+/>
+
+### 2. Create Function Decoder
+
+Next, we'll be creating a Function decoder to decode the payload from the device
+and properly format it for the API endpoint. The decoder is specific to the
+encoding scheme used to encode the data before the device transmits it. For off
+the shelf devices this is usually manufacture specific. For development devices,
+[CayenneLPP](https://developers.mydevices.com/cayenne/docs/lora/#lora-cayenne-low-power-payload)
+if often used. For this guide we'll be using a decoder specifically made for
+this tracking device and slightly modifying it by adding two additional required
+field. The original decoder can be found from the manufacture's website
+[here](https://www.dragino.com/downloads/downloads/LGT_92/Decoder/LGT92-v1.5.0_decoder_20191129.txt).
+
+To start, navigate to the Functions page using the left navigation and then
+click _Create New Function_.
+
+1. Enter a name for the decoder, we've used _Dragino LGT92 Decoder_, select the
```

Comment:
Enter a name for the decoder (below we've used _Dragino LGT92 Decoder), select the..